### PR TITLE
Add operator= for JSON type to be used by ANALYZE

### PR DESCRIPTION
**Pull request by a1exsh**

### CODE REVIEW

File: sql/data_collection_helpers/bloated_tables_and_indexes.sql [hunk 1]
```diff
@@ -194,4 +194,41 @@ BEGIN
...
+BEGIN
+    EXECUTE l_jsoneq_text;
+    EXECUTE 'ALTER FUNCTION zz_utils.jsoneq(JSON, JSON) OWNER TO postgres';
+    EXECUTE 'REVOKE EXECUTE ON FUNCTION zz_utils.jsoneq(JSON, JSON) FROM public';
+
+    EXECUTE 'DROP OPERATOR CLASS IF EXISTS zz_utils.json_ops USING HASH';
+    EXECUTE 'DROP OPERATOR IF EXISTS zz_utils.= (JSON, JSON)';
+
+    EXECUTE 'CREATE OPERATOR zz_utils.= (PROCEDURE=zz_utils.jsoneq, LEFTARG=JSON, RIGHTARG=JSON, HASHES, MERGES)';
+    EXECUTE 'CREATE OPERATOR CLASS zz_utils.json_ops DEFAULT FOR TYPE JSON USING HASH AS OPERATOR 1 zz_utils.=';
```
Comment (a1exsh) [inline sql/data_collection_helpers/bloated_tables_and_indexes.sql]: Do we really have to call these through EXECUTE?  Just copying the style here, but I'm not sure I see the point.

Comment (feikesteenbergen) [inline sql/data_collection_helpers/bloated_tables_and_indexes.sql]: I don't see the point either, perhaps a remnant from the past?

Comment (valgog) [inline sql/data_collection_helpers/bloated_tables_and_indexes.sql]: And mark the function as IMMUTABLE STRICT

Comment (kmoppel) [inline sql/data_collection_helpers/bloated_tables_and_indexes.sql]: yeah no point for EXECUTE here - copy-paste, wohoo. @a1exsh could you please change it also then

**Comment by a1exsh** [comment]
This is far from being optimal.  Given that a proper fix was committed to 9.5, I think we better avoid the workaround altogether.

**Pull request closed without merging**